code/Test_definitions/Traffic_Influence.yaml

```diff
@@ -0,0 +1,1030 @@
+    to end users that are nearby, as the network path is shorter. A network
+    operator's EdgeCloud may comprise multiple Edge Cloud Zones, each in a
+    discrete location to bring latency benefits to end users across a country.\
+    The operator can help developers know which of the Edge Cloud Zones will
+    bring the best performance benefit for a given end user and application\.
+    The Traffic Influence API (TI API) provides the fastest routing from the
+    user Device (e.g. a Smartphone) to the optimal EAS instance in a specific
+    geographical location, installed in an Edge Cloud Zone.\
+    If a Service is offered by Cloud Instances and by Edge Instances, the TI API
+    can be used get the optimal routing of the traffic to the Edge Instances,
+    maybe for a set of users. Getting the optimal routing can be used to improve
+    latency maybe in combination with other CAMARA APIs such as QoD (Quality On
+    Demand). Providing the optimal routing is indeed an important step to get
+    the optimal latency.\
+    If the TI API is used to get the best routing at the Edge for a Device in a
+    geographical location and the Device moves to another geographical location,
+    the TI API can be invoked to get the optimal routing in the new geographical
+    location for that Device.
+    ## Introduction
+    The TI API provides the capability to establish the best routing, in terms
+    of latency, in a specific geographical area, between the user Device, e.g.
+    the user’s smartphone, and the optimal EAS instance nearby. If the Device
+    moves in a different geographical location where a more suitable EAS
+    instance is available, the TI API can be invoked to influence the Device
+    connectivity to get the optimal routing to the that local instance. It is i
+    mportant to notice that it is a task of the Application invoking the TI API
+    to detect the changes in the Device location.\
+    The generic architecture for the Service can foresee some Cloud instances of
+    the Application, one or more Edge Instances of the Application. A component
+    of the Service is the TI API Consumer. This logical component can be
+    integrated in other components of the Service to invoke the TI API, creating
+    a "TrafficInfluence" resource specifying the desired intent.\
+    The TI API Producer implements the intent specified in the
+    "TrafficInfluence" resource.\
+    While the TI API can be invoked to activate the fastest routing for any
+    user, it can also be used to request the best routing for a specific user.
+    Invoking the TI API for each user, many "TrafficInfluence" resources are
+    created for each user to provide the requested routing for a set of users.\
+    The same approach is used for the geographical locations where the influence
+    of the traffic must be applied. Invoking the TI API without specifying a
+    geographical area activates the optimal routing toward any EAS instance,
```

Comment:
I think without specifying a geographical area the optimal routing should be towards a platform determined EAS instance which could possibly provide the optimal routing for a user in a given location rather than "any EAS"? Any thoughts?